packages/Microsoft.Bot.Components.Telephony/Microsoft.Bot.Components.Telephony.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version Condition=" '$(PreviewPackageVersion)' == '' ">$(LocalPackageVersion)</Version>
```

Comment:
These build variables exist inthe SDK but not in this repo. Check out the graph csproj to be consistent in how versioning works here.

Reply:
ok I took it from the Teams csproj will now look at the graph one.